Xamarin.Essentials/Launcher/Launcher.ios.tvos.cs

```diff
@@ -30,47 +31,27 @@ static Task<bool> PlatformTryOpenAsync(Uri uri)
         }
 
 #if __IOS__
+
         static UIDocumentInteractionController documentController;
 
         static Task PlatformOpenAsync(OpenFileRequest request)
         {
-            var fileUrl = NSUrl.FromFilename(request.File.FullPath);
-
-            documentController = UIDocumentInteractionController.FromUrl(fileUrl);
-            documentController.Delegate = new DocumentControllerDelegate
+            documentController = new UIDocumentInteractionController()
```

Comment:
What is the difference between the `ctor()` or `UIDocumentInteractionController.FromUrl`? Is there a reason?

Reply:
https://github.com/xamarin/Essentials/issues/1592#issuecomment-751392147

I think it works a little bit faster

Reply:
I saw the delay on the sim, but only on the first run. When I clicked it again, it was fast. But both work so it is ok.

Reply:
actually, i noticed the delay on my device too... seems like a bug in iOS as we have already sent things over. It can't be our file as we just create on with bytes. So, i think the OS has to load all the providers.